test/camunda-cloud/ElementTemplatesReplaceProviderSpec.js

```diff
@@ -0,0 +1,340 @@
+import {
+  inject,
+  getBpmnJS,
+  bootstrapCamundaCloudModeler
+} from 'test/TestHelper';
+
+import {
+  query as domQuery
+} from 'min-dom';
+
+import {
+  isTemplateApplied
+} from 'lib/camunda-cloud/features/replace/ElementTemplatesReplaceProvider';
+
+import simpleXml from '../fixtures/replaceElementTemplates.bpmn';
```

Comment:
Let's move test diagrams and implementation close to each other.

Name `ElementTemplatesReplaceProvider.element-templates.json`
Name `ElementTemplatesReplaceProvider.bpmn`.

This way we embrace locality (and ensure diagrams are not accidentally used by others).

Reply:
I'll incorporate this.

Reply:
I know we have `fixtures` in many places still, but it is an anti pattern (to move things away from each other).

Reply:
Good example is bpmn-js, where we stared with a HUGE ball of fixtures, and now moved to a very "focused per test" fixture set. That worked much better.